changes/8626.feature

```diff
@@ -1,3 +1,5 @@
 :ref:`ckan.user.unique_email_states` can be used to specify statuses of user
 accounts that are used for checking uniqueness of the email during
-registration.
+registration. After changing the value of the option, run `ckan db
+duplicate_emails` CLI command to verify that all existing emails are still
+unique.
```

Comment:
The most important part of this change is treating differing email cases as the same, so even if you haven't touched the new option you'll still want to run the duplicate_emails cli check, right?

Reply:
Yes, makes sense. I created a separate changelog record for case-insensitive emails and mentioned there recommendation to run the command